package.json

```diff
@@ -46,12 +66,13 @@
     "mocha-lcov-reporter": "^1.3.0",
     "nyc": "^15.0.0",
     "prettier-standard": "^16.4.1",
-    "standard": "^16.0.0"
+    "rimraf": "^3.0.2",
+    "standard": "^16.0.3",
+    "ts-standard": "^10.0.0",
+    "typescript": "^4.1.3"
   },
   "engines": {
     "node": ">=12.0.0"
   },
-  "dependencies": {
-    "typescript": "^4.1.3"
```

Comment:
That means that the typescript runtime dep was wrongly there before the TS rewrite :)